src/pymovements/dataset/dataset_library.py

```diff
@@ -33,6 +33,7 @@ class DatasetLibrary:
     ----------
     definitions:
         Dictionary of :py:class:`~pymovements.DatasetDefinition`.
+
```

Comment:
i'm confused, is this needed? I thought some of the docstring linters should complain about empty lines at the end?

Reply:
hmm, don't know, actually it should -- I'll check